lib/services/arweave/arweave_service.dart

```diff
@@ -161,19 +197,31 @@ class ArweaveService {
   ///
   /// returns DriveEntityHistory object
   Future<DriveEntityHistory> createDriveEntityHistoryFromTransactions(
-      List<DriveEntityHistory$Query$TransactionConnection$TransactionEdge>
-          queryEdges,
-      SecretKey? driveKey,
-      String? owner,
-      int lastBlockHeight) async {
-    final entityTxs = <
-        DriveEntityHistory$Query$TransactionConnection$TransactionEdge$Transaction>[];
-
-    entityTxs.addAll(queryEdges.map((e) => e.node).toList());
+    List<DriveEntityHistory$Query$TransactionConnection$TransactionEdge$Transaction>
+        entityTxs,
+    SecretKey? driveKey,
+    String? owner,
+    int lastBlockHeight, {
+    required SnapshotDriveHistory snapshotDriveHistory,
+  }) async {
+    final responses = await Future.wait(
```

Comment:
Be explicit on the type
```suggestion
    final List<Uint8List> responses = await Future.wait(
```

Reply:
Applied - b90b79ba7cfd6cbe4edc4f171a242895ba664183